### PR TITLE
added Order

### PR DESCRIPTION
**Pull request by roman680**

### CODE REVIEW

File: src/main/java/com/project/bookstore/service/OrderServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        BigDecimal total = shoppingCartFromDB.getCartItems().stream()
+                .map(c -> c.getBook().getPrice().multiply(BigDecimal.valueOf(c.getQuantity())))
+                        .reduce(BigDecimal.ZERO, BigDecimal::add);
+        order.setStatus(Order.Status.PENDING);
+        order.setUser(user);
+        order.setShippingAddress(requestDto.getShippingAddress());
+        order.setTotal(total);
+        orderRepository.save(order);
+        Set<CartItem> cartItems = shoppingCartFromDB.getCartItems();
+        order.setOrderItems(setOrderItems(cartItems, order));
```
Comment (Rommelua) [inline src/main/java/com/project/bookstore/service/OrderServiceImpl.java]:
```suggestion
       setOrderItems(cartItems, order);
```

File: src/main/java/com/project/bookstore/service/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        return orderMapper.toResponseDto(orderRepository.save(order));
```
Comment (Rommelua) [inline src/main/java/com/project/bookstore/service/OrderServiceImpl.java:68]: What is with the shoppingCartFromDB now? Is it empty?

File: src/main/java/com/project/bookstore/repository/OrderItemRepository.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
+package com.project.bookstore.repository;
+
+import com.project.bookstore.model.OrderItem;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (sokimaaa) [inline src/main/java/com/project/bookstore/repository/OrderItemRepository.java]:
```suggestion
```

File: src/main/java/com/project/bookstore/repository/OrderRepository.java [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package com.project.bookstore.repository;
+
+import com.project.bookstore.model.Order;
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.stereotype.Repository;
+
+@Repository
```
Comment (sokimaaa) [inline src/main/java/com/project/bookstore/repository/OrderRepository.java]:
```suggestion
```

**Pull request merged**